### PR TITLE
[liquidity redesign] Update liquidity empty state design (base)

### PR DESCRIPTION
**Pull request by Diogomartf**
### WIP 
_Me and @berteotti are working on updating the design of liquidity page and also the fonts and colors._

This is **work in progress** it's not finished, but we want to divide the work in smaller chunks and do things incrementally. 

We will do this smaller PRs and merge it on a "main" branch, `feature/fonts-and-colors-liquidity-page`.

--- 

## Description

Add empty state base design.

image
 
# To Test

1. <<Step one>> Open liquidity page
- [ ] If there is no pools it should look like the design above.

### CODE REVIEW

File: src/components/Pool/PairsList/index.tsx [hunk 1]
```diff
@@ -141,3 +143,39 @@ export default function PairsList({ aggregatedPairs, loading, filter, singleSide
...
+
+  background: linear-gradient(143.3deg, rgba(46, 23, 242, 0.5) -185.11%, rgba(46, 23, 242, 0) 49.63%);
+  background: linear-gradient(113.18deg, rgba(255, 255, 255, 0.5) -0.1%, rgba(0, 0, 0, 0) 98.9%);
+  background: #3933584d;
+
+  background-blend-mode: normal, overlay, normal;
+  backdrop-filter: blur(25px);
+`
+
+const BlueButton = styled.button`
```
Comment (berteotti) [inline src/components/Pool/PairsList/index.tsx:162]: I think this button is very similar to `CurrencySelect`, we must create composable components so it makes it easier to edit styles. Maybe we can have a simple BlueButton were we can apply new styles on. Unfortunately we don't have good buttons under `components/Button`, at least for this case.

File: src/components/Pool/PairsList/index.tsx [hunk 2]
```diff
@@ -141,3 +143,39 @@ export default function PairsList({ aggregatedPairs, loading, filter, singleSide
     </Flex>
   )
 }
+
+const DimBgContainer = styled.div`
+  width: 100%;
+
+  padding: 48px;
+  border-radius: 12px;
+  border: 1px solid #464366;
```
Comment (berteotti) [inline src/components/Pool/PairsList/index.tsx:152]: We should use theme colors here

File: src/components/Pool/PairsList/index.tsx [hunk 3]
```diff
@@ -141,3 +143,39 @@ export default function PairsList({ aggregatedPairs, loading, filter, singleSide
...
+
+  background: linear-gradient(143.3deg, rgba(46, 23, 242, 0.5) -185.11%, rgba(46, 23, 242, 0) 49.63%);
+  background: linear-gradient(113.18deg, rgba(255, 255, 255, 0.5) -0.1%, rgba(0, 0, 0, 0) 98.9%);
+  background: #3933584d;
+
+  background-blend-mode: normal, overlay, normal;
```
Comment (berteotti) [inline src/components/Pool/PairsList/index.tsx:158]: Should we use this background colors? Shouldn't this be a simple blur?

File: src/components/Pool/PairsList/index.tsx [hunk 4]
```diff
@@ -119,11 +118,14 @@ export default function PairsList({ aggregatedPairs, loading, filter, singleSide
...
-            <Text fontSize="12px" fontWeight="700" lineHeight="15px" letterSpacing="0.08em">
-              NO PAIRS YET
-            </Text>
-          </Empty>
+          <DimBgContainer>
+            <Flex alignItems="center" justifyContent="center" flexDirection={'column'}>
+              <Text fontSize="16px" color="#BCB3F0" mb="24px">
+                No pools found
+              </Text>
+              <BlueButton>Create a pool</BlueButton>
```
Comment (berteotti) [inline src/components/Pool/PairsList/index.tsx:126]: Don't forget translations

Comment (Diogomartf) [reply src/components/Pool/PairsList/index.tsx:126]: I'll add it.

File: src/components/Pool/PairsList/index.tsx (hunk 3 above)
Comment (Diogomartf) [reply src/components/Pool/PairsList/index.tsx:158]: I don't know what you mean by a "simple blur". I used Figma to get the opacity, gradients, of the component.

File: src/components/Pool/PairsList/index.tsx (hunk 1 above)
Comment (Diogomartf) [reply src/components/Pool/PairsList/index.tsx:162]: Currently refactoring the components in another PR.

File: src/components/Pool/PairsList/index.tsx [hunk 5]
```diff
@@ -50,18 +33,15 @@ interface PairsListProps {
...
 export default function PairsList({ aggregatedPairs, loading, filter, singleSidedStake }: PairsListProps) {
   const { chainId } = useActiveWeb3React()
   const [page, setPage] = useState(1)
   const responsiveItemsPerPage = useResponsiveItemsPerPage()
   const itemsPage = usePage(aggregatedPairs, responsiveItemsPerPage, page, 0)
   const { address: swprAddress } = useSWPRToken()
   const { loading: loadingNativeCurrencyUsdPrice, nativeCurrencyUSDPrice } = useNativeCurrencyUSDPrice()
+  const { t } = useTranslation()
+
   // const [layoutSwitch, setLayoutSwitch] = useState<Layout>(Layout.LIST)
```
Comment (berteotti) [inline src/components/Pool/PairsList/index.tsx:45]: Can you delete this line? Otherwise I think it's ready to merge

Comment (berteotti) [inline src/components/Pool/PairsList/index.tsx:45]: Can you delete this line? Otherwise I think it's ready to merge

```suggestion
```

**Review by berteotti — APPROVED** [review]

**Pull request merged**